app/controllers/report_controller/reports/editor.rb

```diff
@@ -1295,6 +1299,7 @@ def set_form_vars
 
       # @edit[:new][:cb_include_metrics] = nil - it means YES (YES is default value for new and legacy reports)
       @edit[:new][:cb_include_metrics] = options[:include_metrics].nil? || options[:include_metrics]
+      @edit[:new][:method_for_allocated_metrics] = options[:method_for_allocated_metrics].try(:to_sym) || DEFAULT_CHARGEBACK_ALLOCATED_METHOD
```

Comment:
`_()` needs to be applied to the values of the `CHAREGEBACK_ALLOCATED_METHODS` hash for i18n
to really work here (same in the rest of this PR).